metadata_please/tests/sdist.py

```diff
@@ -90,11 +93,29 @@ def test_basic_metadata_absl_py_09(self) -> None:
         )
         self.assertEqual({"test"}, bm.provides_extra)
 
+    def test_basic_metadata_fields(self) -> None:
```

Comment:
can you include a couple more words about who does this -- is this from modern setuptools, or something else like flit/poetry/hatchling

Reply:
Update test to have Description.